docs/source/markdown/podman-machine-start.1.md

```diff
@@ -45,6 +47,51 @@ Start the specified podman machine.
 ```
 $ podman machine start myvm
 ```
+Output:
+
+Starting machine "myvm"
+This machine is currently configured in rootless mode. If your containers
+require root permissions (e.g. ports < 1024), or if you run into compatibility
+issues with non-podman clients, you can switch using the following command:
+
+	podman machine set --rootful myvm
+
+API forwarding listening on: /var/folders/my/k1kpd8ds7p938mchyb1r_4mw0000gn/T/podman/myvm-api.sock
+
+The system helper service is not installed; the default Docker API socket
+address can't be used by podman. If you would like to install it, run the following commands:
+
+        sudo /opt/homebrew/Cellar/podman/5.2.4/bin/podman-mac-helper install
+        podman machine stop myvm; podman machine start myvm
+
+You can still connect Docker API clients by setting DOCKER_HOST using the
+following command in your terminal session:
+
+        export DOCKER_HOST='unix:///var/folders/my/k1kpd8ds7p938mchyb1r_4mw0000gn/T/podman/myvm-api.sock'
+
+Machine "myvm" started successfully
+
+Start with --no-info:
+```
+$ podman machine start --no-info myvm
+```
+Output:
+
+Starting machine "myvm"
+Machine "myvm" started successfully
+
+
+Start with --quiet:
+```
+$ podman machine start --quiet myvm
+```
+Output:
+
+Machine "myvm" started successfully
+
```

Comment:
The output is unreadable, please check the rendered markdown to see how it looks. The correct way we include the output is to just copy it into the same code block as the command, i.e. simply
```
$ podman machine start --no-info myvm
Starting machine "myvm"
Machine "myvm" started successfully
```